Sources/StreamChat/Controllers/ChannelController/ChannelController+SwiftUI_Tests.swift

```diff
@@ -23,7 +23,7 @@ class ChannelController_SwiftUI_Tests: iOS13TestCase {
         
         XCTAssertEqual(observableObject.state, channelController.state)
         XCTAssertEqual(observableObject.channel, channelController.channel)
-        XCTAssertEqual(observableObject.messages, channelController.messages)
+        XCTAssertEqual(Array(observableObject.messages), Array(channelController.messages))
```

Comment:
We use our own `XCTAssertEqual` overload. Isn't there a way how to solve this by introducing some specific overload so we don't have to cast it everywhere?

Reply:
Half solved by making collection to conform to `Equatable`. Solves cases when you compare one Lazy to another Lazy.
Cases when you compare it to other datatype (Array) are to rare to bother right now.